compat/src/forwardRef.js

```diff
@@ -24,17 +24,10 @@ export const REACT_FORWARD_SYMBOL =
  * @returns {import('./internal').FunctionComponent}
  */
 export function forwardRef(fn) {
-	// We always have ref in props.ref, except for
-	// mobx-react. It will call this function directly
-	// and always pass ref as the second argument.
-	function Forwarded(props, ref) {
-		let clone = Object.assign({}, props);
+	function Forwarded(props) {
+		let clone = assign({}, props);
```

Comment:
Minor nit: I think we switched to `Object.assign` in `main` and don't have the `assign()` helper anymore.